### DISCUSSION TITLE
Set margins and font size by default for epub documents?

### DISCUSSION THREAD

**Discussion by nobody98765** [koreader/koreader#9888] [Q&A] [answered]
Is there a way to set margins and font size **by default** for all epub documents? 

What I mean is I need to set margins and font-size manually every time when opening a new epub file. **I'd like to make it so that every epub that's opened inherits the same font-size and margins.**

**Comment by Frenzie** [comment, marked as answer]
Long press on the setting to set default. That should work for just about all settings.

More complicated, there's also a plugin to set different default document settings per folder under tools → more tools → tweak document settings.

**Reply by nobody98765** [reply]
Thank you very much - I had seen the long-press somewhere else but didn't think to try it for the book settings in KOreader. Thanks again.
